src/kibana/components/agg_types/controls/percentile_ranks.html

```diff
@@ -0,0 +1,9 @@
+<div class="form-group">
+  <label>Values</label>
+  <kbn-number-list
+    ng-model="agg.params.values"
+    unit-name="value"
+    range="[0,Infinity]"
```

Comment:
Does this need to be restricted at 0? Theoretically, can't a percentile rank be executed for any value?